Exercise_D and E/index.html

```diff
@@ -7,16 +7,53 @@
     <meta name="description" content="" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="stylesheet" href="./assets/style/index.css" />
-    <link
-      rel="stylesheet"
-      href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
-      integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh"
-      crossorigin="anonymous"
-    />
+  
+    
   </head>
   <body>
     <!-- CODE HERE -->
+    <nav class="navbar">
+        <div class="navbar-header">
+          <img src="./assets/images/logo.png">
+        </div>    
+    </nav>
+    
+    <section class="home" id="home">
+      <div class="max-width">
+          <div class="home-content">
+              <h1>Beautiful Flowers</h1>
+              <a href=# class="btn">Learn more</a>
+          </div>
+      </div>
+    </section>
 
+    <div class = "container">
+      <div class="card">
```

Comment:
Please remember to add a value to the alt props on your html elements. Leaving it blank means if there is an issue loading the image, your users will now know what is meant to be there.